### DISCUSSION TITLE
提个问，目前很需要建立分区吗？

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#1821] [其他]
建立分区这个事儿从原理上看，想曲线救国不是那么复杂，利用标签的特性似乎就可以做到，但是有了分区就肯定要出现版主来管理分区。

如果一个板块没有版主，或许会出现以下这些情况：
**1、缺乏管理和组织**：版主通常负责维护讨论区的秩序，如标记答案、管理论坛内容和删除不适当的内容。没有版主，这些任务可能无法有效执行，导致讨论区混乱无序。

**2、信息过载**：缺乏版主可能导致信息过载，用户难以在大量讨论中找到有用的信息。

**3、不当内容的风险**：没有版主来监管，讨论区可能会出现不当或无关的内容，影响社区的整体氛围。

当然靠社区的自主管理似乎也是可以的。不过，根据举报处那一堆举报消息，我认为还是有一定的风险的。当然，做这个项目的话，那又要辛苦up那边了，本来项目就多，如果这个也要做的话，那又要再制作一个组件了。

**Comment by cpu644** [comment]
开发团队？一个大佬加上一堆摸鱼提意见的（乐

**Reply by nostalgiatan** [reply]
就俩人，就他俩在那里贡献

**Comment by whoisfucker** [comment]
游戏里的绳网只是服务于游戏的
现实里的网站如果没有分区，论坛的规模就算变大，也只是看到的内容更随机而已，没有利用好规模的优势

当然分区是一种方式，还有标签（tag，分类、词云）的方式，甚至用搜索也是一种服务于同一目的相关方案

**Reply by whoisfucker** [reply]
早期可以先考虑做搜索，因为这样方便知道用户在搜什么，然后建立分区也会更简单

**Reply by nostalgiatan** [reply]
> 游戏里的绳网只是服务于游戏的
> 现实里的网站如果没有分区，论坛的规模就算变大，也只是看到的内容更随机而已，没有利用好规模的优势
> 
> 当然分区是一种方式，还有标签（tag，分类、词云）的方式，甚至用搜索也是一种服务于同一目的相关方案

或许可以先确定标签词条，然后发公告引导用户发在对应的标签下面也算是一种分区了吧？
不过UI上面好像只有一个推送功能，也就是说还需要一个可以切换分区的按钮。

**Reply by whoisfucker** [reply]
如果一个人想看关于“派派”的角色的攻略，那么他可能要点击“攻略”、“角色”，点击“阵营”或属性或异常角色等，最后才能定位到派派的信息

因此我建议先推出搜索，再推出分区，可以是左上角有b站类似的搜索栏（如果要减少ui占用，可以默认是一个按钮，点击后再出现搜索）

另一件重要的一件事是，传统的分区（像B站的分区）对于论坛来说其实不简单，一旦定型就很难修改，可以说分区就是给网站定型，里面的分区罗列顺序，以及各个分区的定义和行为约束、重要性分析都是一门学问

所以基于“话题”、词条或自定义标签的分区，可能更简单，并且可以随着趋势修改
当然用户最好可以给分区添加主封面图片，让每个分区看起来像B站的分区一样

如果分区不是强制的，或有对应的激励（流量激励或创作激励、趣味激励），其实没有多少人愿意分区

**Reply by nostalgiatan** [reply]
受教了(／(ｴ)＼)
